spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/Convertor.scala

```diff
@@ -69,8 +69,12 @@ object BytesToMat {
 class PixelBytesToMat(byteKey: String = ImageFeature.bytes) extends FeatureTransformer {
 
   override def transformMat(feature: ImageFeature): Unit = {
+    require(feature.getOriginalSize != null,
+      "please set the original size of image in ImageFeature")
     val pixels = feature[Array[Byte]](byteKey)
-    val mat = OpenCVMat.fromPixelsBytes(pixels, feature.getOriginalHeight, feature.getOriginalWidth)
+    val mat = OpenCVMat.fromPixelsBytes(pixels, feature.getOriginalHeight,
```

Comment:
why not using feature.getOriginalSize._1 feature.getOriginalSize._2 to keep consistent with the way channel is fetched

Reply:
Or maybe add a new method called getChannel ?